.github/workflows/workflow.yaml

```diff
@@ -16,17 +16,11 @@ jobs:
       - name: Check out code into the Go module directory
         uses: actions/checkout@v2
 
-      - name: Install golangci-lint
-        run: |
-          mkdir -p $GITHUB_WORKSPACE/bin
-          echo "$GITHUB_WORKSPACE/bin" >> $GITHUB_PATH
-          curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $GITHUB_WORKSPACE/bin v${GOLANGCILINT_VERSION}
-        env:
-          GOLANGCILINT_VERSION: 1.35.2
-
-      - name: Make lint
-        run: |
-          make lint
+      - name: Lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.35.2
```

Comment:
did you wanted to bump this to `v1.38.0` as PR title suggests?

Reply:
@sozercan good catch thx